.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The `.vscode` folder is `gitignored` but looks like this file is committed before doing so (so probably you want to run `git rm -f .vscode/settings.json` or thereabouts so that you're not proposing adding it in this PR.